src/seam-connect/client.ts

```diff
@@ -1,13 +1,27 @@
-import axios, { AxiosInstance, AxiosRequestConfig } from "axios"
+import axios, {
+  AxiosInstance,
+  AxiosRequestConfig,
+  AxiosRequestHeaders,
+  AxiosResponse,
+} from "axios"
 import axiosRetry from "axios-retry"
 import { SeamAPIError, SeamMalformedInputError } from "../lib/api-error"
 import { Routes } from "./routes"
-import { ErroredAPIResponse, SuccessfulAPIResponse } from "../types/globals"
+import {
+  APIResponse,
+  ErroredAPIResponse,
+  SuccessfulAPIResponse,
+} from "../types/globals"
 import { version } from "../../package.json"
+import { ClientSession, ClientSessionResponse } from "../types"
 
 export interface SeamClientOptions {
   /* Seam API Key */
   apiKey?: string
+  /* Seam Client Access Token */
+  clientAccessToken?: string
```

Comment:
`clientAccessToken` is mislabeled, `seam_at` should never be used on a client. Maybe `personalAccessToken`?

Reply:
I think this is just the old name right? Should be `clientSessionToken`?